tools/php-cs/.styleci.yml

```diff
@@ -0,0 +1,23 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+#
+# Package `sllh/php-cs-fixer-styleci-bridge` is required to get it working.
+
+preset: symfony
+
+enabled:
+  - align_double_arrow
```

Comment:
I understand but it was a @rande request. :stuck_out_tongue: 

For now I'll keep as is to get it working. Feel free to open a PR once this one merged to start a discussion about it. :+1: